docs/cinder-csi-plugin/multi-region-clouds.md

```diff
@@ -0,0 +1,316 @@
+# Multi Az/Region/Openstack Configuration
+
+### Multi cluster Configuration file
+
+Create a configuration file with a subsection per openstack cluster to manage (pay attention to enable ignore-volume-az in BlockStorage section).
+
+Example of configuration with 3 regions (The default is backward compatible with mono cluster configuration but not mandatory).
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloud-config
+  namespace: kube-system
+type: Opaque
+stringData:
+  cloud.conf: |-
+    [BlockStorage]
+    bs-version=v3
+    ignore-volume-az=True
+    
+    [Global]
+    auth-url="https://auth.cloud.openstackcluster.region-default.local/v3"
+    username="region-default-username"
+    password="region-default-password"
+    region="default"
+    tenant-id="region-default-tenant-id"
+    tenant-name="region-default-tenant-name"
+    domain-name="Default"
+    
+    [Global "region-one"]
+    auth-url="https://auth.cloud.openstackcluster.region-one.local/v3"
+    username="region-one-username"
+    password="region-one-password"
+    region="one"
+    tenant-id="region-one-tenant-id"
+    tenant-name="region-one-tenant-name"
+    domain-name="Default"
+    
+    [Global "region-two"]
+    auth-url="https://auth.cloud.openstackcluster.region-two.local/v3"
+    username="region-two-username"
+    password="region-two-password"
+    region="two"
+    tenant-id="region-two-tenant-id"
+    tenant-name="region-two-tenant-name"
+    domain-name="Default"
+```
+
+
+
+### Create region/cloud secrets
+
+Create a secret per openstack cluster which contains a key `cloud` and as value the subsection's name of corresponding openstack cluster in configuration file.
```

Comment:
I would still use the correct terminology in everywhere in this PR. You call openstack region as "openstack cluster". It will confuse users in long run. Could you modify terminology from "openstack cluster" -> "openstack region".